### ISSUE TITLE
graphviz is not installed in any of osx Travis builds

### ISSUE THREAD

**Issue by aalexand** [google/pprof#317]
See https://travis-ci.org/google/pprof/jobs/338682508 build.

It has
```
$ export SKIP_GRAPHVIZ=0
```

But none of the brew commands seem to be executed, and the test prints out
```
--- PASS: TestWebInterface (0.03s)
	webui_test.go:87: skipping / since dot (graphviz) does not seem to be installed
```

This is unexpected.

**Comment by aalexand** [comment]
The build output https://travis-ci.org/google/pprof/jobs/309160817 which is from #270 seems to have the same issue, so apparently it was broken from the start.

**Comment by nolanmar511** [comment]
I'm assuming I broke this with PR#27 (which was supposed to only skip graphviz installation on xcode6.4)

**Comment by nolanmar511** [comment]
Fixed with #318
